verification/curator-service/api/src/model/parser.ts

```diff
@@ -3,8 +3,7 @@ import mongoose from 'mongoose';
 export const parserSchema = new mongoose.Schema({
     awsLambdaArn: {
```

Comment:
Is it worth changing this field to awsLambdaName?

Reply:
I think it's going to break all existing parsers, if we're fine with that then sure, but maybe in another PR.